packages/notification-service/src/firebase.ts

```diff
@@ -91,8 +92,15 @@ export function initializeDb() {
   lastBlockRef.on(
     'value',
     (snapshot) => {
-      console.debug('Latest block data updated: ', snapshot && snapshot.val())
-      lastBlockNotified = (snapshot && snapshot.val()) || 0
+      const lastBlock = (snapshot && snapshot.val()) || 0
+      console.debug('Latest block data updated: ', lastBlock)
+      if (lastBlockNotified < 0) {
+        // This is to prevent sending again notifications for transfers that happened in the previous |MAX_BLOCKS_TO_WAIT|
+        // blocks when deploying this service. We might miss some notifications when deploying.
+        lastBlockNotified = lastBlock + MAX_BLOCKS_TO_WAIT
```

Comment:
No, when we query we substract the 120 blocks again, so we should get everything after the last notified block stored on Firebase. If there were any blocks we didn't process before that block before the deploy, we'd miss those since we're not looking for previous blocks anymore. I think it should be a small number of blocks we might miss.